### PR TITLE
Dt 159

### PR DESCRIPTION
**Pull request by siren**

### CODE REVIEW

File: menu.sh [hunk 1]
```diff
@@ -137,6 +138,10 @@ function selectAction {
             printAction "Building pelias"
             ansible-playbook -i environments/$ENV -K -s playbooks/build-pelias.yaml -u $USERNAME
             ;;
+        "16")
+            printAction "ldigitraffic2gtfsrt"
```
Comment (samuliheljo) [inline menu.sh]: what is that 1 or l in front of digitraffic2gtfsrt

File: roles/otp-docker-image/templates/router-config.json.j2 [hunk 2]
```diff
@@ -39,6 +39,22 @@
       "sourceType": "kml",
       "url": "http://routeserver:{{route_server_exposed_port}}/citybikes.kml",
       "frequencySec": -1
+    },
+    {
+      "type": "stop-time-updater",
+      "frequencySec": 60,
+      "sourceType": "gtfs-http",
+      "url": "http://raildigitraffic2gtfsrt:{{raildigitraffic2gtfsrt_exposed_port}}/national",
+      "defaultAgencyId": "JOLI",
```
Comment (samuliheljo) [inline roles/otp-docker-image/templates/router-config.json.j2]: What should this be? "JOLI" is for tampere? ping @hannesj

File: variables/common.yaml [hunk 3]
```diff
@@ -82,6 +82,11 @@ vector_map_server_exposed_port: 8089
 vector_map_server_style_git: https://github.com/HSLdevcom/osm-bright.tm2source.git
 vector_map_server_style: osm-bright.tm2source
 
+raildigitraffic2gtfsrt_root: "{{docker_build_root}}/raildigitraffic2gtfsrt"
+raildigitraffic2gtfsrt_docker_image: "{{docker_image_base}}/raildigitraffic2gtfsrt"
+raildigitraffic2gtfsrt_exposed_port: 8090
+raildigitraffic2gtfsrt_git: https://github.com/pailakka/raildigitraffic2gtfsrt
```
Comment (samuliheljo) [inline variables/common.yaml:88]: should we fork this under hsldevcom?

File: roles/otp-docker-image/templates/router-config.json.j2 (hunk 2 above)
Comment (hannesj) [inline roles/otp-docker-image/templates/router-config.json.j2]: This is currently "MATKA" on dev.digitransit.fi

File: variables/common.yaml (hunk 3 above)
Comment (hannesj) [inline variables/common.yaml:88]: Good question, we should check this

Comment (samuliheljo) [inline variables/common.yaml:88]: let's leave it like this for now

**Pull request merged**